From 7f2e2b47b7c13f4c7bf0cf5a61b1e7981579dce5 Mon Sep 17 00:00:00 2001
From: rapour <reza.abbasalipour@canonical.com>
Date: Wed, 29 Oct 2025 18:30:45 +0400
Subject: [PATCH] fix ignore_root_user_error

Signed-off-by: rapour <reza.abbasalipour@canonical.com>
---
 WORKSPACE | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/WORKSPACE b/WORKSPACE
index 6439f904..f7003252 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -77,7 +77,7 @@ envoy_dependencies()
 
 load("@envoy//bazel:repositories_extra.bzl", "envoy_dependencies_extra")
 
-envoy_dependencies_extra()
+envoy_dependencies_extra(ignore_root_user_error=True)
 
 load("@envoy//bazel:python_dependencies.bzl", "envoy_python_dependencies")
 
-- 
2.43.0
